ADotNet.sln

```diff
@@ -14,6 +14,10 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{16CB1804-3040-41F0-9C1F-99A139B24873}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
+		.gitattributes = .gitattributes
```

Comment:
What does this do?

Reply:
This adds the `.gitattributes` file to the "Solution Items" folder in Visual Studio:

![image](https://user-images.githubusercontent.com/507649/227834996-a4a1b510-31cf-4834-babd-b2f704aea1c3.png)

The `.gitattributes` file is already checked into the repo and is currently configured to normalize line endings.

You can learn more about `.gitattributes` here: https://git-scm.com/docs/gitattributes